### DISCUSSION TITLE
New HCaptcha

### DISCUSSION THREAD

**Discussion by LeoTuet** [LeoTuet/ui-we-want-to-sleep#100] [General]
Due to accessibility we want to limit the use of the `HCaptcha`

**Comment by marcschwaiger** [comment]
Using HCaptcha might prevent people from using it (think about the last time you had to solve a captcha, it hurts. + Some people are simply not able to solve captchas. Even slow internet speeds might ruin the user experience)

**Comment by marcschwaiger** [comment]
TODO: Create a test endpoint, to check if proxies are detected correctly
